src/main/java/net/dancier/chatdancer/ChatCreatedResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package net.dancier.chatdancer;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.UUID;
+@Data
+public class ChatCreatedResponseDto {
```

Comment:
I am not finished with Dtos, I just used that because I was trying to get the values from the RequestBody and I had issues. I will create the appropriate Dto for every case (Requests and Responses)